igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/artifact/decorator/DebDetailsDecorator.groovy

```diff
@@ -41,7 +41,8 @@ class DebDetailsDecorator implements ArtifactDetailsDecorator {
         if (!genericArtifact.fileName) {
             return false
         }
-        return genericArtifact.fileName.tokenize('.').last() == "deb"
+        return (genericArtifact.fileName.tokenize('.').last() == "deb" &&
+          genericArtifact.fileName.tokenize(versionDelimiter).size() >= 3)
```

Comment:
could you throw a comment in here that the deb file naming convention is package_version_architecture just so there is a bit of context why we are requiring 3 components

Reply:
Done!